contracts/Baal.sol

```diff
@@ -322,11 +320,10 @@ contract Baal is Module, EIP712Upgradeable, ReentrancyGuardUpgradeable, BaseRela
                 expiration > block.timestamp + votingPeriod + gracePeriod,
             "expired"
         );
+        require(baalGas < 30000000 gwei, "baalGas to high"); /* eth block limit */
```

Comment:
from G0 feedback

> 6) There's an interesting DoS vector where proposer can set a very high prop.baalGas to make a proposal unprocessable. If this proposal will have more "yes" votes than "no" votes by the time voting ends, regardless of whether or not it reaches quorum, this proposal will basically jam the DAO making execution of any further proposals impossible.

hard coding an upper limit to of 30mm baalGas will keep someone from using something (on purpose or accidentally) which could cause a deadlock if passed